authors/filip-srdoc/15-porezi/hr_HR/final.md

```diff
@@ -0,0 +1,107 @@
+Kada bi fizička osoba primjerice rudarila bitcoin kod kuće za vlastite potrebe, to bi se smatralo drugim dohotkom koji inače ulazi u sustav sintetičkog oporezivanja dohotka zajedno s dohotkom od nesamostalnog rada i samostalne djelatnosti. Ako ukupni ostvareni prihod ne prelazi 210 000 kn porezna stopa je 24%, a ukoliko prelazi porezna stopa se penje na 36%.
+Kriptovaluta stvorena rudarenjem za privatne potrebe bi se tako moglo smatrati dohotkom od samostalne djelatnosti, a s obzirom na to mogla bi je svrstati pod definiciju “obrta izjednačenih djelatnosti”. Tako bi primici bili izrudarena kriptovaluta, a odbici računalo koje rudari.
+
+### Prodaja kriptovaluta
+
+Kad bi osoba u sklopu svoje poslovne aktivnosti prodala izrudareni bitcoin taj poslovni primitak bio bi oporeziv dohotkom od samostalne djelatnosti. No, proda li fizička osoba izrudareni bitcoin izvan svoje poslovne djelatnosti nameće se više kategorija dohotka za klasifikaciju. 
+Pošto se u većini zemalja kriptovaluta smatra _assetom_ odnosno imovinom, pretpostavljalo bi se da onda dohodak od te imovine spada u kategoriju dohotka od imovine i imovinskih prava. U Hrvatskoj pod to spadaju najamnine, zakupnine, prihodi od iznajmljivanja stanova i soba i slično, ali ne i kriptovalute. Kriptovalute bi mogli svrstati pod kapitalna dobra, a dohotke od njih pod dohotke od kapitala tj. kapitalne dobitke. Kapitalna dobit je razlika prodajne i nabavne cijene financijske imovine. Prema Poreznoj upravi Republike Hrvatske:
+
+> "Trgovanje kriptovalutama smatra se financijskom transakcijom te se slijedom prethodno navedenog, na dohodak ostvaren po osnovi trgovanja kriptovalutama plaća porez na dohodak po osnovi kapitalnih dobitaka, budući se radi o dobitku po osnovi kupoprodaje te valute, što je ekvivalent instrumentima tržišta novca."
+
+Ukoliko je porezni obveznik te ostvario gubitak od prodaje bitcoina, on ga može isključivo odbiti od drugih kapitalnih dobitaka u toj godini, pa makar to bila dobit od prodaje dionica. Poznavajući volatilno tržište kriptovaluta, gubici su itekako mogući pa je zbog toga moguće da si porezni obveznik neće moći smanjiti poreznu osnovicu poreza na dohodak.
+
+Ako tvrtka prodaje kriptovalute ili ako fizička osoba koja plaća porez na dohodak prijeđe u sustav poreza na dobit, tada se na prihode od prodaje kriptovaluta primjenjuje porez na dobit. Time prihodi od prodaje postaju poslovni prihodi, a ne kapitalni pa se postavlja pitanje što je isplativije. Primjerice, poslovni gubitak se može prenositi pet godina dok se kapitalni gubitak može odbiti od kapitalnog dobitka jedino u tekućoj godini. Porezna stopa poreza na dobit u Hrvatskoj iznosi 18% odnosno 12% ukoliko u poreznom razdoblju ostvareni prihodi nisu prešli 3 000 000 kn. To znači da porezna stopa ovisi o visini prihoda pa tako ona može biti jednaka ili viša od porezne stope kapitalnog dobitka u sklopu sustava poreza na dohodak. Odluka o boljoj opciji je na kraju krajeva individualna i ovisi o okolnostima subjekta.
+
```

Comment:
A što ako se prodaje izrudareni bitcoin? U tom slučaju nema ulazne cijene. Treba taj scenarij pokrit.

Reply:
to je opisano jedino djelomično za ostale države, ali ne i za Hrvatsku. Ali kaćžu da je po tom pitanju hrv slicna sloveniji pa sam napisao ovo.